packages/kit/src/exports/node/polyfills.js

```diff
@@ -16,7 +16,13 @@ const globals = {
 };
 
 // exported for dev/preview and node environments
+// TODO: remove this once we only support Node 18.11+ (the version multipart/form-data was added)
 export function installPolyfills() {
+	// don't polyfill on non-Node platforms like Deno
+	if (!globalThis.process?.versions?.node) {
+		return;
+	}
```

Comment:
Hey folks, Bartek from the Deno team here. This fix won't work - when executing code from npm packages we are injecting globals like `process` or `setImmediate` behind the scenes, if we didn't do that most of packages wouldn't work as they often rely on `process` global. I suggest to do `typeof globalThis.Deno !== "undefined"` check instead

Reply:
The error reported in https://github.com/sveltejs/kit/pull/8338#issuecomment-1372057526 is caused by this check

Reply:
@bartlomieju thanks! I had tested `process.versions.node` directly and not through an npm package. I'm curious, what do you set `process.versions.node` to? I wonder if my earlier solution of polyfilling only when the version is less than 18.11.0 might work for Deno